### PR TITLE
Registering on new NAIP bucket 

### PR DESCRIPTION
**Pull request by abhiataero**
Registering the new bucket for NAIP data.

### CODE REVIEW

File: datasets/naip.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  -	multispectral
+  - sustainability
+License: |
+  Public Domain with Attribution
+Resources:
+  - Description: Imagery in MRF format 4-band (RGB + NIR) and metadata in a Requester Pays bucket
+    ARN: arn:aws:s3:::naip-analytic
+    Region: us-west-2
+    Type: S3 Bucket
+- Description: Original Imagery in Geotiff format 4-band (RGB + NIR) uncompressed no overviews and metadata in a Requester Pays bucket
```
Comment (jflasher) [inline datasets/naip.yaml]: Needs to be indented.

File: datasets/naip.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ARN: arn:aws:s3:::naip-source
+    Region: us-west-2
+    Type: S3 Bucket
+- Description: Visualization Imagery in Cloud Optimized Geotiff format 3-band (RGB) Compressed with overviews and metadata in a Requester Pays bucket
```
Comment (jflasher) [inline datasets/naip.yaml]: Needs to be indented.

File: datasets/naip.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Name: NAIP on AWS
+Description: The National Agriculture Imagery Program (NAIP) acquires aerial imagery during the agricultural growing seasons in the continental U.S. This “leaf-on” imagery andtypically ranges from 60 centimeters to 100 centimeters in resolution. More details [here](https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/).
+Documentation: https://docs.opendata.aws/naip-analytic/readme.html
+Contact: https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/
+UpdateFrequency: NAIP data is delivered at the state level and every year number of states receive updates with an overall update cycle of three years.
+Tags:
+  - aws-pds
+  - earth observation
+  - satellite imagery
```
Comment (jflasher) [inline datasets/naip.yaml]: Please remove `satellite imagery` (incorrect) and `multispectral` (I don't think we'll want that as a tag) and add `aerial photography` to https://github.com/awslabs/open-data-registry/blob/master/tags.yaml in this PR.

Comment (abhiataero) [reply datasets/naip.yaml]: @jflasher 

- Fixed syntax 
- Added the Tag in tags.yaml, used aerial imagery 
- Removed multispectral and satellite imagery

**Pull request merged**